stripper/ze_dangerous_waters_v2_Final.cfg

```diff
@@ -1,3 +1,45 @@
+;Make the laser secret not useless, since damage value on env_beam and env_laser is damage per second, not per damage tick.
+modify:
+{
+	match:
+	{
+		"classname" "env_laser"
+		"targetname" "Lazer"
+	}
+	replace:
+	{
+		"damage" "40000"
+	}
+}
+
+;Make it so watermelon gun does not stop CTs mid-jump to prevent trolling teammates
+add:
+{
+	"classname" "trigger_multiple"
+	"targetname" "L_Melon_trig"
+	"origin" "-8923 -5704 -1830"
+	"wait" "0"
+	"StartDisabled" "0"
+	"spawnflags" "1"
+	"filtername" "Human Filter"
+	"parentname" "L_Melon"
+	"model" "*155"
+	"OnStartTouch" "L_Melon,RunScriptCode,activator.SetOwner(self),0,-1"
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "point_template"
+		"targetname" "temp"
+	}
+	insert:
+	{
+		"Template02" "L_Melon_trig"
+	}
+}
```

Comment:
This seems like a weird fix, unless I'm misunderstanding something. Wouldn't it make more sense to have the OnStartTouch output here just break L_Melon? Parenting it to the player to prevent collision just seems odd.

Reply:
I tried that way out first, but then it broke immediately due to the trigger touching the player that is actually using the item. This way also allows it to go through other CTs and still stop zombies mid jump (which is how it currently behaves, but also messes up cts), as the melon doesn't break on contact with a player.

Reply:
Oh alright, didn't realize that's what you were doing.